lineman/app/models/base_model.coffee

```diff
@@ -5,13 +5,14 @@ angular.module('loomioApp').factory 'BaseModel', ->
     @indices: []
     @attributeNames: []
 
-    constructor: (recordsInterface, data) ->
+    constructor: (recordsInterface, data, postInitializeData = {}) ->
       @errors = {}
       @processing = false
       Object.defineProperty(@, 'recordsInterface', value: recordsInterface, enumerable: false)
       Object.defineProperty(@, 'recordStore', value: recordsInterface.recordStore, enumerable: false)
       Object.defineProperty(@, 'restfulClient', value: recordsInterface.restfulClient, enumerable: false)
       @initialize(data)
+      _.merge @, postInitializeData
```

Comment:
Here it wasn't setting up views for the clone because it had no id at the time, so have to set the id on the model before we finish the initialize method.